networking/hardware_networks/about-sriov.adoc

```diff
@@ -17,6 +17,14 @@ The SR-IOV network device driver for the device determines how the VF is exposed
 
 You can use SR-IOV network devices with additional networks on your {product-title} cluster installed on bare metal or {rh-openstack-first} infrastructure for applications that require high bandwidth or low latency.
 
+You can configure multi-network policies for SR-IOV networks. The support for this is technology preview and SR-IOV additional networks are only supported with kernel NICs. They are not supported for DPDK applications.
```

Comment:
Does this need a Technology Preview admonition box with all of the relevent support statements?

I assume that these are in place where they need to be (perhaps out-of-scope for this review), but I just want to double check

Reply:
Unlike SR-IOV and NIC, the ISG, SSG, and repo glossary do not list _DPDK_ as an acronym that does not need to be spelled out. 
```suggestion
You can configure multi-network policies for SR-IOV networks. The support for this is technology preview and SR-IOV additional networks are only supported with kernel NICs. They are not supported for Data Plane Development Kit (DPDK) applications.
```

Reply:
@michaelryanpeter Yes I was not sure if tech preview snippet was required. I should probably add it as you say.